notebooks/249-oneformer-segmentation/249-oneformer-segmentation.ipynb

```diff
@@ -14,7 +14,7 @@
    "id": "b6dbb8aa-c070-4e7d-8b7a-64588e7f0da4",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #25.            plt.close("all")
</pre><p>Do you really need this line here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1363/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Reply:
It's not strictly required, but otherwise there is a warning about too many figures open at the same time

Reply:
<p>OK, I found this weird to see during a performance estimation. Maybe it's better to move this line after the calculation inference time for the sample?</p>

Reply:
Decided to remove it altogether, because memory footprint of opened figures is tiny compared to memory required to run the whole notebook.